src/Frontend/Core.hs

```diff
@@ -240,6 +249,42 @@ instance Page p => Page (Frame p) where
     isAuthorized a = isAuthorized (_frameBody <$> a)
     extraPageHeaders = extraPageHeaders . _frameBody
 
+instance (Page a, Page b) => Page (a :<|> b) where
+    isAuthorized = error "IMPOSSIBLE"
+
+instance (KnownSymbol s, Page a) => Page (s :> a) where
+    isAuthorized = error "IMPOSSIBLE"
+
+instance (KnownSymbol s, Page a) => Page (Capture s c :> a) where
+    isAuthorized = error "IMPOSSIBLE"
+
+instance Page a => Page (Headers h a) where
+    isAuthorized = error "IMPOSSIBLE"
+
+instance (KnownSymbol s, Page a) => Page (QueryParam s b :> a) where
+    isAuthorized = error "IMPOSSIBLE"
+
+instance Page a => Page (FormReqBody :> a) where
+    isAuthorized = error "IMPOSSIBLE"
+
+instance Page a => Page (Get c a) where
+    isAuthorized = error "IMPOSSIBLE"
+
+instance Page a => Page (Post c a) where -- TODO
```

Comment:
is there a reason why `Get` above doesn't get a `TODO`, but `Post` and `PostResult` do?